benchmark.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+const path = require('path');
+const Benchmark = require('benchmark');
+const makeDir = require('make-dir');
+const tempy = require('tempy');
+const del = require('.');
+
+const suite = new Benchmark.Suite('concurrency');
+
+const tempDir = tempy.directory();
+
+const fixtures = Array.from({length: 2000}, (x, index) => {
+	return path.resolve(tempDir, (index + 1).toString());
+});
+
+function createFixtures() {
+	for (const fixture of fixtures) {
+		makeDir.sync(path.resolve(tempDir, fixture));
+	}
+}
+
+const concurrency = [1, 3, 5, 10, 15, 20, 50, 100, Infinity];
+
+concurrency.forEach(num => {
```

Comment:
I prefer `for-of` loops whenever possible. Also don't use abbreviations `num` => `number`.

Actually, this should be:
```suggestion
concurrencies.forEach(concurrency => {
```

Reply:
Updated